src/shared/constants.h

```diff
@@ -165,3 +165,6 @@ static const unsigned NUM_TICKS_PER_TRAJECTORY_ELEMENT = 4u;
 static const unsigned CONTROL_LOOP_HZ                  = 60u;
 
 static const unsigned NUM_GENEVA_ANGLES = 5;
+
+static const double ROBOT_COLLISION_BUFFER             = 0.005;
+static const double ROBOT_MAX_STATIONARY_SPEED_M_PER_S = 0.01;
```

Comment:
Constants that are specific to Python should just be define in Python. This way also if we change them, then a bunch of C++ code doesn't have to recompile

Reply:
Ideally we wouldn't mix test and robot constants together, so I agree with nima